calico-cloud_versioned_docs/version-3.15/reference/resources/globalalert.mdx

```diff
@@ -24,7 +24,7 @@ can be used to specify the resource type on the CLI:
 
 ## Sample YAML
 
-```yaml
+```yaml noValidation
```

Comment:
Does this break the validator? It looks fine, and it passes on yamlchecker.com.

I see more than a few of these. Is there something I'm missing?

Reply:
Yes Chris. I added the `noValidation` tag to some of the yaml blocks that were actually correct according to the [tool](https://codebeautify.org/yaml-validator/) I was using. However, I added the tag because the validator was complaining.

Reply:
Removed the tag for testing.

Reply:
Test passed without the tag :).